src/generics/utils.js

```
@@ -874,12 +874,26 @@ const generateDateRanges = (startEpoch, endEpoch, interval) => {
 
 				break
 			case 'month':
-				nextDate = new Date(currentDate)
-				nextDate.setMonth(currentDate.getMonth() + 1)
+				let rangeStart
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)